users/api/requests.go

```diff
@@ -62,6 +63,9 @@ func (req listUsersReq) validate() error {
 	if len(req.email) > maxEmailSize {
 		return apiutil.ErrEmailSize
 	}
+	if req.state != "all" && req.state != "active" && req.state != "inactive" {
```

Comment:
Create consts for these status.  Maybe even change them by `enabled and `disabled`?